dlpy/tests/test_gpu.py

```diff
@@ -0,0 +1,539 @@
+            self.s.retrieve('table.dropcaslib', message_level='error', caslib=caslib)
+                
+    def test_gpu08(self):
+        '''
+        gpu=Gpu(use_tensor_rt=True, use_exclusive=True) in Model.fit() and Model.predict().
+            use_tensor_rt : bool, optional
+                Enables using TensorRT for fast inference.
+                Default: False.
+            use_exclusive : bool, optional
+                Specifies exclusive use of GPU devices.
+                Default: False
+        '''
+        model1 = self.CreateSimpleCNN1()
+        
+        caslib, tmp_caslib, data, inputs, target = self.LoadEEE()
+
+        r = model1.fit(data=data, inputs=inputs, target=target, n_threads=16, mini_batch_size=8, lr=1e-4, 
+                gpu=Gpu(use_tensor_rt=True, use_exclusive=True))
+        #'WARNING: TensorRT only supports inference and will be disabled.'
+        self.assertTrue(r.severity == 1)
+
+        # useTensorRT=True does not support multi-GPU in Viya 4.0.1.
+        # Alireza is working on a fix to print out the warning message when there are multiple
+        # GPU devices available AND no devices= specification. As of 05/21/2020, mva-vbviya
+        # dev20004, model1.predict(data=data, gpu=Gpu(use_tensor_rt=True)) would fail. With
+        # Alireza's fix, the command will complete and print out a warning message.
+        r2 = model1.predict(data=data, gpu=Gpu(devices=[0], use_tensor_rt=True, use_exclusive=True))
+        self.assertTrue(r2.severity == 0)
+
+        if (caslib is not None) and tmp_caslib:
+            self.s.retrieve('table.dropcaslib', message_level='error', caslib=caslib)
+
+    def test_gpu09(self):
+        '''
+        gpu=Gpu(precision='FP16', use_exclusive=True) in Model.fit() and Model.predict().
+            precision : string, optional
+                Specifies the experimental option to incorporate lower computational
+                precision in forward-backward computations to potentially engage tensor cores.
+                Valid Values: FP32, FP16
+                Default: FP32
+            use_exclusive : bool, optional
+                Specifies exclusive use of GPU devices.
+                Default: False
+        '''       
+        model1 = self.CreateSimpleCNN1()
+        
+        caslib, tmp_caslib, data, inputs, target = self.LoadEEE()
+
+        # Default lr=1e-2 causes FPE if precision='FP16'.
+        r = model1.fit(data=data, inputs=inputs, target=target, n_threads=16, mini_batch_size=8, lr=1e-4, 
+                gpu=Gpu(precision='FP16', use_exclusive=True))
+        self.assertTrue(r.severity == 0)
+
+        r2 = model1.predict(data=data, 
+                gpu=Gpu(precision='FP16', use_exclusive=True))
+        self.assertTrue(r2.severity == 0)
+
+        if (caslib is not None) and tmp_caslib:
+            self.s.retrieve('table.dropcaslib', message_level='error', caslib=caslib)
+
+    # Skip the test until Alireza fixes TensorRT issues.
```

Comment:
Please remove.  If the unit test will not run as-is, then simply comment it out.